### DISCUSSION TITLE
About Karaoke-style movie generation

### DISCUSSION THREAD

**Discussion by ashyrv** [ggml-org/whisper.cpp#1206] [General]
Hey everyone! I have a question about the karaoke-style speech recognition. Do you think it will work in real time? It seems it's really good with recorded audio but will it work in real-time speech? Any feedback or answers appreciated. 

Thank you in advance!

**Comment by ulatekh** [comment]
Whisper is not real-time, nor is it likely to ever be real-time. It works by processing audio 30 seconds at a time, and that processing, even when GPU-accelerated, can take a significant fraction of a second, or even several seconds.

**Comment by ashyrv** [comment]
Thanks
